movie-search.js

```diff
@@ -18,9 +21,10 @@ const printTitles = (rawData) => {
   titles.forEach((element) => {
     console.log(element)
   })
+  return titles
```

Comment:
Your `printTitles` function is doing two things:
1. parsing the titles out of the HTML
2. displaying those titles to the user (via console.log)

Your test is just testing the return value of this function, so it's really only testing the logic for the first thing. If you comment out the console.log your test will still pass because you're not actually testing what gets printed. It's actually probably OK not to test the `console.log` since that logic is so simple, but it's a design smell that your function is doing two things and the awkwardness of the test is pointing to that. If I were you I would rename this function to `parseTitlesFromHTML` and just have it return the titles without printing them. Then your tests cover all the functionality. Then move the `console.log` down to where `printTitles` (now `parseTitlesFromHTML`) is being called. Like this:

```js
try {
      const titles = parseTitlesFromHTML(rawData)
      titles.forEach(console.log)
    } catch (error) {
      console.error(error)
    }
```